dashboard/of-cloud-dashboard/handler.js

```diff
@@ -3,6 +3,33 @@
 const fs = require('fs');
 const request = require('request');
 
+const handleLogout = (context) => {
+  const now = new Date();
+  const year = now.getFullYear();
+  const month = now.getMonth();
+  const day = now.getDate();
+  const expires = new Date(year, month, day);
+  const headers = {
+    'Set-Cookie': [
+      'openfaas_cloud_token=',
+      `Expires=${expires.toUTCString()}`,
+      `Domain=${process.env.cookie_root_domain}`,
+      'Path=/',
+    ].join('; ')
+  };
+
+  fs.readFile(`${__dirname}/dist/logout.html`, (err, data) => {
+    if (err) {
+      context.status(500).fail(err);
```

Comment:
You need a return statement here otherwise it will fall through into the next block also.